### PR TITLE
add more sample data

### PR DESCRIPTION
**Pull request by lilozz2**
Added more people to SampleData, currently at 20

### CODE REVIEW

**Review by KamiliArsyad — APPROVED** [review]
hm probably gonna clash a lot with Dick's PR

**Review by MagnificentCreature — APPROVED** [review]
lgtm, thanks for clearing the clashes

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -76,7 +76,175 @@ public static Person[] getSamplePersons() {
...
+                        .addInteraction(new Interaction(
+                                "Asked for CRM features", Interaction.Outcome.valueOf("INTERESTED"),
+                                LocalDate.of(2023, 10, 15)
+                        ))
+                        .addInteraction(new Interaction(
+                                "Requested a pricing plan", Interaction.Outcome.valueOf("FOLLOWUP_REQUIRED"),
+                                LocalDate.of(2023, 10, 22)
+                        ))
+                        .build(),
+
```
Comment (zhyuhan) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: Shall we get rid of the empty lines to maintain consistency?

**Review by zhyuhan — APPROVED** [review]
LGTM!

**Pull request merged**